README.md

```diff
@@ -7,101 +7,59 @@
 
 ## Manuscript description
 
-To see what's incoming, check the [open pull requests](https://github.com/greenelab/deep-review/pulls).
+This repository is home to **The Deep Review**, a collaboratively written review article on deep learning in precision medicine.
+The preprint for this study, titled **Opportunities and obstacles for deep learning in biology and medicine**, is [available on _bioRxiv_](https://doi.org/10.1101/142760).
+The Deep Review is collaboratively written on GitHub using a tool called Manubot (see [below](#manubot)).
+The project operates on an open contribution model, welcoming contributions from anyone (see [`CONTRIBUTING.md`](CONTRIBUTING.md) or an [existing example](https://github.com/greenelab/deep-review/pull/147 "Pull request 147: Hardware Limitations and Scaling") for more info).
+To see what's incoming, check the open [pull requests](https://github.com/greenelab/deep-review/pulls).
+For project discussion and planning see the [Issues](https://github.com/greenelab/deep-review/issues).
 
-#### Current Status - Submitted
+### Current stage: revisions
 
-We have submitted the manuscript to the journal. A preprint is available at
-[bioRxiv](https://doi.org/10.1101/142760). We are still accepting contributions
-to improve this work. Feel free to create an issue, contribute some text via a
-pull request, and pitch in. Authorship criteria remain the same.
+We are currently working on manuscript revisions in response to two external peer reviews.
+See [Issue #678](https://github.com/greenelab/deep-review/issues/678) to coordinate this process.
+When addressing reviewer comments, please also update [`response-to-reviewers.md`](content/response-to-reviewers.md) as appropriate.
```

Comment:
Do you want to make this stronger? We'd like to not accept PRs on new topics unless they directly address review comments? Basically just "may" -> "will"